### PR TITLE
Consistency and documentation / endpoint work

### PR DESCRIPTION
**Pull request by CreeperBeatz**
yes

### CODE REVIEW

**Review by CreeperBeatz — COMMENTED** [review]
Damn that code is clean! 👍 
image

File: README.md [hunk 1]
```diff
@@ -121,64 +122,72 @@ You can register/login in the app using our internal protocol, or OAuth2 and goo
...
+      [ # Only if employees are assigned 
+        {
+        "added_by": int,
+        "can_reward": bool,
+        "email": str,
+        "first_name": str,
+        "last_name": str,
+        "place_id": int,
+        "profile_picture": str 
+        },
```
Comment (KirilMihaylov) [inline README.md]: JSON doesn't allow comma as last character and this is more explicit that it's a list.
```suggestion
        }, ...
```

File: README.md [hunk 2]
```diff
@@ -191,61 +200,146 @@ str,
...
-            "given_on": str,
-            "place_id": int,
-            "visitor_id": int
-        }
+      {
+        "employee_id": int,
+        "given_on": str,
+        "place_id": int,
+        "visitor_id": int
+      }
```
Comment (KirilMihaylov) [inline README.md]: Same as above.
```suggestion
      }, ...
```

File: src/app.py [hunk 3]
```diff
@@ -132,54 +133,109 @@ async def get_site_info():
...
+        401: not authorized
+        400: expired/invalid token
+        400: employee trying to give himself a stamp
+        400: already have this stamp
+    """
     async with async_session() as session:
-        stamp: Stamp = finish_stamping(token, g.authenticated_user)
+
+        # Make a stamp object
+        stamp_token: str = (await request.form).get('stamp_token')
```
Comment (KirilMihaylov) [inline src/app.py:217]: Pretty sure that either returns `None` or raises an exception like that.

File: src/app.py [hunk 4]
```diff
@@ -335,18 +413,18 @@ async def self_info():
         {long json info}
 
     excepts:
-        401 - not authorized
+        401 : not authorized
```
Comment (KirilMihaylov) [inline src/app.py]: Not consistent with other end-points.

**Review by KirilMihaylov — COMMENTED** [review]
Some minor things, but overall looks good.

File: src/app.py (hunk 3 above)
Comment (CreeperBeatz) [reply src/app.py:217]: Actually it works perfectly

**Pull request merged**